lib/active_storage/service/bunny_service.rb

```diff
@@ -8,13 +8,16 @@ module ActiveStorage
   # See ActiveStorage::Service for the generic API documentation that applies to all services.
   class Service::BunnyService < Service
 
-    CDN_BASE_URL = "https://#{ENV['BUNNY_STORAGE_ZONE']}.b-cdn.net"
-
     attr_reader :client
```

Comment:
make base_url as a reader method to keep consistency with `ActiveStorage::Service` abstract class or any implemented class like `ActiveStorage::Service::S3Service`